### ISSUE TITLE
Error showing modules list in Linux Mint 18.3 x64

### ISSUE THREAD

**Issue by wertyy102** [trimstray/sandmap#64]
After setting up sandmap following the directions, this is what happens when I try to load module list.

![image](https://user-images.githubusercontent.com/26676249/40760281-11a4a95c-6464-11e8-89df-4ed53657c918.png)
also when i go to exit the shell session using #exit or $exit after running sandmap, this happens.
![image](https://user-images.githubusercontent.com/26676249/40760310-36d5a5dc-6464-11e8-8173-f00098a2c534.png) in a scrolling fashion.

**Comment by trimstray** [comment]
Hi!

First of all: paste the log files:

- log/sandmap.<date>.log
- log/stdout.log

I will try to reproduce this error on mint.

**Comment by wertyy102** [comment]
Will do, I'll be home from work shortly and get those up!

**Comment by wertyy102** [comment]
[sandmap.log](https://github.com/trimstray/sandmap/files/2065501/1vRsGp5d.txt)
and stdout.log is completely and entirely empty...  

Sorry for the delay, got caught up in some work stuff.

**Comment by trimstray** [comment]
@wertyy102 : fix included in 60ae657.

Before running please execute this:
```bash
git pull origin master && git fetch --all && git fetch --prune --tags
```

@Vincentgking : thx!
